src/main/resources/application.conf

```diff
@@ -29,6 +29,7 @@ etc-client {
       max-blocks-headers-per-message = 200
       max-blocks-bodies-per-message = 200
       max-receipts-per-message = 200
+      max-mpt-components-per-message = 400
```

Comment:
it depends how we want to organize configs. based on functionality (fast-sync) or the place where this config is used